docs/configurations/index.md

```diff
@@ -56,12 +57,16 @@ Currently available tab components:
             "entity": []
         },
         {
-            "name": "proxy",
-            "title": "Proxy",
-            "entity": [],
-            "options": {
-                "saveValidator": ""
-            }
+            "type": "proxyTab",
+             "warning": {
+                "config": {
+                    "message": "Some warning message"
+                }
+            },
+            "proxy_type": true,
+            "username": true,
+            "password": true,
```

Comment:
Does it make sense to move `proxy_type`, `username` and `password` to a default configuration? I feel that more TAs would benefit from having it already there.

Reply:
With respect to proxy server setup, `username`, `password` and `proxy_type` are not required configurations. I am inclined not keeping by default so that developers have awareness of the new features UCC rolls out. Yet, giving these fields by default would easy for the developers.